### PR TITLE
App theme updates

### PR DESCRIPTION
**Pull request by kildre**
Added updates to the Nebari theme. Amending colors to use themeing colors and configuartion. Updated logo to a CDAO | Nebari logo
## Reference Issues or PRs
https://gitlab.jatic.net/jatic/team-metrostar/t-e-platform/-/issues/669

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [x ] Code style update (formatting, renaming)
- [ x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

## Any other comments?
image
image

image
image

### CODE REVIEW

File: jhub_apps/templates/style.css [hunk 1]
```diff
@@ -422,10 +422,6 @@ a:focus {
     outline-color: var(--secondary-color) !important;
 }
 
-.MuiButton-textPrimary {
-    color: var(--primary-color) !important;
-}
-
```
Comment (jbouder) [inline jhub_apps/templates/style.css:428]: This  should not be removed, it affects other things.

File: jhub_apps/templates/style.css [hunk 2]
```diff
@@ -449,11 +445,6 @@ a:focus {
 .MuiInputLabel-root {
     color: var(--text-color) !important;
 }
-
-.MuiButton-textPrimary#profile-menu-btn {
-    color: var(--navbar-text-color) !important;
-}
-
```
Comment (jbouder) [inline jhub_apps/templates/style.css:456]: This should not be removed

File: jhub_apps/themes/__init__.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 from jhub_apps.version import get_version
 
-LOGO = "/services/japps/static/img/Nebari-Logo-Horizontal-Lockup-White-text.svg"
+# LOGO = "/services/japps/static/img/Nebari-Logo-Horizontal-Lockup-White-text.svg"
```
Comment (jbouder) [inline jhub_apps/themes/__init__.py:3]: This can be removed

File: ui/src/components/navigation/navigation.tsx [hunk 4]
```diff
@@ -378,9 +379,27 @@ export const TopNavigation = ({ ...props }): React.ReactElement => {
...
+                  backgroundColor: theme.palette.gray[50],
+                },
+              }}
             >
-              {currentUser?.name} {currentUser?.admin ? '(admin)' : ''}
+              {currentUser?.name}{' '}
+              {currentUser?.admin && (
+                <Chip
+                  label="admin"
+                  // color="primary"
```
Comment (jbouder) [inline ui/src/components/navigation/navigation.tsx:394]: Remove this

**Review by jbouder — CHANGES_REQUESTED** [review]
In addition to the comments, a few other things:
1. Remove the CDAO logo
2. These changes do no provide a mechanism to override the header colors based on what is provided as a config. You'll need to update this file to handle those: `jhub_apps/static/templates/style.css`

File: jhub_apps/templates/style.css (hunk 1 above)
Comment (kildre) [reply jhub_apps/templates/style.css:428]: Returned styles

File: jhub_apps/templates/style.css (hunk 2 above)
Comment (kildre) [reply jhub_apps/templates/style.css:456]: Returned styles

File: jhub_apps/themes/__init__.py (hunk 3 above)
Comment (kildre) [reply jhub_apps/themes/__init__.py:3]: Removed

File: ui/src/components/navigation/navigation.tsx (hunk 4 above)
Comment (kildre) [reply ui/src/components/navigation/navigation.tsx:394]: Removed

File: jhub_apps/templates/style.css [hunk 5]
```diff
@@ -22,12 +22,13 @@
     --link-text-color: {{ text_color | default("#1c1d26") }};
     --primary-color: {{ primary_color | default("#ba18da") }};
     --primary-color-dark: {{ primary_color_dark | default("#9b00ce") }};
+    --primary-color-light: {{ primary_color_light | default("#ba18da10") }};
```
Comment (jbouder) [inline jhub_apps/templates/style.css:25]: I see you added this, but its not being used anywhere. Also, if planning to use this, Nebari does not currently provide primary_color_light as a variable, so if you need this we should discuss adding it to Nebari as well.

File: jhub_apps/templates/style.css [hunk 6]
```diff
@@ -274,6 +276,9 @@ a:focus {
     background-color: {{ navbar_hover_color | default("#00a3b0") }};
 }
 
+.MuiButtonBase-root.MuiListItemButton-root {
+}
+
```
Comment (jbouder) [inline jhub_apps/templates/style.css:281]: This doesn't appear to be doing anything, remove if not.

File: jhub_apps/themes/__init__.py [hunk 7]
```diff
@@ -7,16 +7,17 @@
     "logo": LOGO,
     "favicon": FAVICON,
     "primary_color": "#ba18da",
+    "primary_color_light": "#BA18DA10",
```
Comment (jbouder) [inline jhub_apps/themes/__init__.py:10]: Same comment as above

File: jhub_apps/themes/__init__.py [hunk 8]
```diff
@@ -7,16 +7,17 @@
...
     "primary_color_dark": "#9b00ce",
     "secondary_color": "#18817a",
     "secondary_color_dark": "#12635e",
     "accent_color": "#eda61d",
     "accent_color_dark": "#a16d14",
-    "text_color": "#1c1d26",
+    "text_color": "#f00ad1",
```
Comment (jbouder) [inline jhub_apps/themes/__init__.py:16]: Changing the text color to pink?

File: jhub_apps/themes/__init__.py [hunk 9]
```diff
@@ -7,16 +7,17 @@
...
     "h1_color": "#0f1015",
     "h2_color": "#0f1015",
-    "navbar_text_color": "#ffffff",
-    "navbar_hover_color": "#20b1a8",
+    "navbar_text_color": "#2E2F33",
+    "navbar_hover_color": "#00000008",
     "navbar_color": "#1c1d26",
```
Comment (jbouder) [inline jhub_apps/themes/__init__.py:21]: If changing the navbar_text_color to black, the navbar_color should probably be a shade of white?

**Review by jbouder — CHANGES_REQUESTED** [review]
In addition to the comments:
- Still need to remove the CDAO logo
- The css provided in the style.css overrides the default JupyterHub header styling, but still doesn't override the MUI header styling. Need to address that.
- I can't seem to test your changes locally. I'm not seeing updates applied to JupyterHub, need to troubleshoot that

File: jhub_apps/templates/style.css (hunk 6 above)
Comment (kildre) [reply jhub_apps/templates/style.css:281]: This was supposed to have the --primary-color-light: {{ primary_color_light | default("#ba18da10") }}; as its styling.

I

**Review by jbouder — APPROVED** [review]
Looks great!

**Pull request merged**